build.gradle

```diff
@@ -25,8 +25,8 @@ dependencies {
 
 group = 'com.cannonhighlight'
 version = '1.0'
-sourceCompatibility = '1.8'
 
-tasks.withType(JavaCompile) {
+tasks.withType(JavaCompile).configureEach {
 	options.encoding = 'UTF-8'
+	options.release.set(11)
```

Comment:
What exactly does the 11 represent here?

Reply:
JDK 11 (runelite now runs with java 11 baseline rather than java 1.8)

Reply:
the official plugin template has the same line: https://github.com/runelite/example-plugin/blob/master/build.gradle#L34